exercises/bob/introduction.md

```diff
@@ -0,0 +1,18 @@
+# Introduction
+
+Bob is a teenager.
+He's not very smart, but he likes to think that he's very cool.
+He definitely doesn't get excited about things.
+That wouldn't be cool.
+
+When people talk to him, his responses are pretty limited.
+
+If you ask him a question, such as "How are you?", Bob answers 'Sure.'
+
+He answers 'Whoa, chill out!' if you YELL AT HIM (in all capitals).
+
+He answers 'Calm down, I know what I'm doing!' if you yell a question at him.
+
+He says 'Fine. Be that way!' if you address him without actually saying anything.
+
+He answers 'Whatever.' to anything else.
```

Comment:
Not saying this would be better, but did you consider making the five different types of responses into a list?